### PR TITLE
Insect Catch Game

### PR DESCRIPTION
**Pull request by Ayush7614**
## Related Issue
  - #166 

"closes: #166"

### Describe the changes you've made
Added a game of insect catching game in html css js

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.
Insect Catch 1

### CODE REVIEW

**Comment by Ayush7614** [comment]
Plss review it @Vishal-raj-1 and this pull is for issue no #166

**Review by urvashi-code1255 — COMMENTED** [review]
@Ayush7614 Correct the paths everywhere!

Also change the card image with GIF

**Comment by Ayush7614** [comment]
@urvashi-code1255 now you can review it

**Comment by urvashi-code1255** [comment]
@Ayush7614 Resolve the merge conflicts!

Write "closes: #166" in the PR description.

File: script.js [hunk 1]
```diff
@@ -180,6 +180,10 @@ let projectData = [
     projectUrl: "public/Weather_app.html",
   },
   {
+    projectName: "Insect Catch Game",
+    projectImage: "assets/Images/Insect Catch.png",
+    projectUrl: "public/InsectCatch.html",
+=======
```
Comment (urvashi-code1255) [inline script.js:186]: @Ayush7614 remove this

**Comment by Ayush7614** [comment]
@urvashi-code1255 now you can see it its correct or not

**Comment by urvashi-code1255** [comment]
@Ayush7614 Its not Okay Yet. Check the website locally.

File: script.js [hunk 2]
```diff
@@ -363,6 +363,14 @@ let projectData = [
     projectUrl: 'public/bookmarks.html',
   },
   {
+    projectName: "Insect Catch Game",
+    projectImage: "assets/Images/Insect Catch.png",
+    projectUrl: "public/InsectCatch.html",
+  },
+    projectName: "Animated Minion Eyes",
```
Comment (urvashi-code1255) [inline script.js:370]: put the open curly brace here

File: script.js [hunk 3]
```diff
@@ -363,6 +363,14 @@ let projectData = [
...
+    projectImage: "assets/GIFs/Minion-Eyes.gif",
+    projectUrl: "public/Minion-Eyes.html",
+  }
     projectName: 'Exchange Rate',
```
Comment (urvashi-code1255) [inline script.js:374]: put the open curly brace here

**Review by urvashi-code1255 — CHANGES_REQUESTED** [review]

**Comment by Vishal-raj-1** [comment]
@Ayush7614 ??

**Comment by Ayush7614** [comment]
@urvashi-code1255 @Vishal-raj-1 now you can review it its correct or not

**Comment by harshgupta20** [comment]
@Ayush7614 Kindly commit on separate branch, pls don't commit on master or main branch.

**Pull request merged**